backend/app/moderation.py

```diff
@@ -441,6 +441,7 @@ def submit_review_request(
                 requests = list(requests)
                 worker.send_email.send(
                     EmailInfo(
+                        message_id=f"{app_id}/{review_request.build_id}/held",
```

Comment:
Worried about `build_id` as that might not be very unique, especially when we get external builds. They are also used in the subject, so that's a problem.